scripts/compose.py

```diff
@@ -2566,7 +2561,11 @@ def start_handler(self):
                     docker_compose_build.append("--no-cache")
                 if args["build_parallel"]:
                     docker_compose_build.append("--parallel")
-                subprocess.call(docker_compose_build + build_services)
+                try:
```

Comment:
LGTM, but could it be possible to reuse code here? We have three instances of same code